HW1/Task2.py

```diff
@@ -0,0 +1,11 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0)
+
+print('Введите трёхзначное число:')
+a = int(input())
+if a < 1000 and a > 99:
+
+    print('Cумма цифр числа', a, ' = ', a % 10 + a % 100 // 10 + a % 1000 // 100)
```

Comment:
Можно чуть иначе и короче
```suggestion
    print('Cумма цифр числа', a, ' = ', a % 10 + a // 10 % 10 + a // 100)
```